docs/pt/docs/advanced/websockets.md

```diff
@@ -0,0 +1,257 @@
+# WebSockets
+
+Você pode usar  <a href="https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API" class="external-link" target="_blank">WebSockets</a> com **FastAPI**.
```

Comment:
There are two spaces here:
Você pode usar  <a href...

Just remove one so it doesn't look strange :D